docs/project/2024/carina.grec/index.md

```diff
@@ -0,0 +1,105 @@
+Data Processing and Logic Unit → LED Controller:
+Depending on the temperature data, the data processing unit sends commands to the LED controller to adjust the LED's color and brightness.
+Data Processing and Logic Unit ↔ Network Interface:
+The network interface acts as a gateway for sending temperature data to the web server and receiving commands from the web application, facilitated by the central data processing unit.
+Network Interface ↔ Web Application:
+The web application communicates with the network interface to send user commands and receive data for display. This includes settings adjustments, real-time temperature updates, and accessing historical data logs.
+
+![LED](https://github.com/carinagrec/upb-fils-ma.github.io/assets/150676040/9fe70bd8-0f70-42c2-90b6-aae9b26a2969)
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Raspberry Pi Pico W - a compact microcontroller board featuring the RP2040 chip and built-in Wi-Fi connectivity. It serves as the brain of the project, managing data collection from the temperature sensor, driving the display, controlling the RGB LED, and handling network communications.
+DS18B20 Temperature Sensor - a digital temperature sensor that provides 9-bit to 12-bit Celsius temperature measurements. Known for its accuracy and ease of integration, it operates over a one-wire bus that requires only one data line (and ground) for communication with the Raspberry Pi Pico W.
+1602 LCD Module - this 16x2 character LCD display is simple and effective for displaying text and numerical data. It is interfaced via the I2C communication protocol, which simplifies the connection by using just two wires for data transmission.
+Common Cathode RGB LED - a small and efficient multi-color LED capable of displaying various colors by mixing red, green, and blue. Each color component can be controlled independently using PWM signals to create different colors.
+Breadboard and Jumper Wires - used for prototyping without soldering, ideal for testing and adjusting the circuit layout during development. Jumper wires facilitate connections between the components on the breadboard.
+Resistors - used for current limiting and voltage division. They are crucial for protecting components like the RGB LED and ensuring correct signal levels are applied to inputs.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|-------|-------|
+| [Raspberry Pi Pico W](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) | The microcontroller with Wi-Fi capability, used for managing sensors and network communication. | [35 RON] |
```

Comment:
In the table, the first link is to wards the components description and the second link (the price) is to the store.